sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc

```diff
@@ -204,11 +219,28 @@ upstream source must provide valid data on every cycle that ready is asserted.
 This is equivalent to changing the pipe read calls to a non-blocking call and assuming that
 success is always true.
 
-If set to `false`, the `min_capacity` pipe class template parameter and `ready_latency`
-property must be 0.
+This property is not guaranteed to be respected if the pipe is an inter-kernel
+pipe. 
+
+This property will only have an effect when the `protocol` property is *avalon_streaming*
+or *avalon_mm*.
+
+|`uses_ready`
+| Ready values: `true` or `false`
+
+Default value: `true`
+
+Controls whether a ready signal is present on the pipe interface. If `false`, the
+downstream sink cannot backpressure the pipe.
+
+This is equivalent to changing the pipe write calls to a non-blocking call and assuming that
+success is always true.
 
 This property is not guaranteed to be respected if the pipe is an inter-kernel
-pipe. The compiler is allowed to optimize the pipe if both sides are visible.
+pipe. 
+
+This property will only have an effect when the `protocol` property is *avalon_streaming*
+or *avalon_mm*.
```

Comment:
This one (and probably some of the others) doesn't make sense for avalon_mm, only streaming.

Reply:
We do have an implementation for device-to-host Avalon MM. I think it does make sense, although I understand that it might not be the preferred or dominant use model.